org.ekstep.mathtext-1.0/editor/mathtext.js

```diff
@@ -12,6 +12,10 @@ angular.module('org.ekstep.mathtext', [])
     $scope.cursorPosition = undefined;
     $scope.text_hint = true;
     $scope.pluginLoadStartTime = new Date();
+    $scope.selectionStart = undefined;
+    $scope.selectionEnd = undefined;
```

Comment:
Remove this $scope.selectionEnd variable

Reply:
Removed check for selectionEnd.